test/e2e/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@okta/e2e",
+  "version": "1.0.0",
+  "main": "runner.js",
+  "private": true,
+  "scripts": {
+    "build": "yarn workspace @okta/okta-signin-widget build:release",
+    "test": "yarn build && node ./runner.js"
+  },
+  "dependencies": {
+    "@okta/okta-auth-js": "*",
```

Comment:
The widget is not always synced with the latest version of auth-js, let's bind auth-js to a specific major version with `^`.